diff --git a/dist/index.js b/dist/index.js
index 323cf865b61fa8b85bac3e166a4fb3489272c6a9..37c4d34fb4c2e5a89601ee5d4a6d9f599ac798b9 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -599,10 +599,14 @@ function VitePluginLibAssets(options = {}) {
       const outputDir = path3.isAbsolute(outDir) ? outDir : path3.posix.join(process.cwd(), outDir);
       const bundleSourceMap = Object.keys(outputBundle).reduce((map, name2) => {
         const filePath = path3.posix.join(outputDir, name2);
-        const source = fs4.readFileSync(filePath, "utf8");
-        map[name2] = source;
+        try {
+          const source = fs4.readFileSync(filePath, "utf8");
+          map[name2] = source;
+        } catch (e) {
+          return false;
+        }
         return map;
-      }, {});
+      }, {});
       const cacheSourceMap = isBuildWatch ? Object.values(Object.fromEntries(assetCache)).reduce((map, { fileName, source }) => {
         if (fileName && source && !bundleSourceMap[fileName])
           map[fileName] = String(source);
