diff --git a/package.json b/package.json
index 4e4c0b203b1650f1657f0f797cecd86c4e9afd42..f370e509ea927a5446cff49a8d76459cea8bfd83 100644
--- a/package.json
+++ b/package.json
@@ -38,7 +38,8 @@
     "tmp": "^0.2.3",
     "vite": "6.3.5",
     "vite-plugin-ejs": "1.7.0",
-    "vite-plugin-static-copy": "3.0.0"
+    "vite-plugin-static-copy": "3.0.0",
+    "reveal.js-mermaid-plugin": "^2.0.0"
   },
   "devDependencies": {
     "@biomejs/biome": "1.9.4",
diff --git a/src/main.js b/src/main.js
index 156c080cf9d188ebdaa2a48cf53718697436f4fe..3dca08332de699fba1e259721f053d1cc654e46d 100644
--- a/src/main.js
+++ b/src/main.js
@@ -5,6 +5,7 @@ import RevealMath from 'reveal.js/plugin/math/math.esm.js';
 import Notes from 'reveal.js/plugin/notes/notes.esm.js';
 import RevealSearch from 'reveal.js/plugin/search/search.esm.js';
 import Zoom from 'reveal.js/plugin/zoom/zoom.esm.js';
+import Mermaid from 'reveal.js-mermaid-plugin/plugin/mermaid/mermaid.esm.js'
 
 import 'reveal.js/dist/reveal.css';
 
@@ -51,7 +52,7 @@ const preConfig = {
 
 const afterConfig = {
 	// Ensure plugins are always loaded and not touched by the theme configuraiton
-	plugins: [Markdown, Highlight, Notes, RevealMath, Zoom, RevealSearch],
+	plugins: [Markdown, Highlight, Notes, RevealMath, Zoom, RevealSearch, Mermaid],
 };
 
 deck.initialize({
